lib/adskHydraSceneBrowser/test/adskHydraSceneBrowserTestFixture.cpp

```diff
@@ -257,7 +257,16 @@ void AdskHydraSceneBrowserTestFixture::CompareValueContent(const pxr::VtValue& v
     std::string actualValue = valueText.toStdString();
 
     std::ostringstream valueStream;
-    valueStream << value;
+    if (value.IsHolding<pxr::SdfPathVector>()) {
```

Comment:
What about other vector types?  E.g. vectors of Plain Old Data (POD) types, like integers, floats?